Vungle/VungleBannerCustomEvent.m

```diff
@@ -118,6 +117,7 @@ - (void)vungleAdWillAppear
 
 - (void)vungleAdDidAppear {
     MPLogInfo(@"Vungle video banner did appear");
+    [self.delegate trackImpression];
```

Comment:
Because we're not able to track clicks currently, we should also remove this and rely on the automatic impression and click tracking.

Reply:
Got it, remove this call. :)